### DISCUSSION TITLE
No more google drive backups? wtf

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [standardnotes/forum#3032] [General]
I just got a message from standard notes 

> We recently tried backing up your data to Google Drive Sync, but an issue prevented us from doing so.
> The usual cause is an expired or revoked token from your sync provider. Please follow [these instructions](https://standardnotes.com/help/27/how-do-i-enable-dropbox-google-drive-or-onedrive-backups) to use CloudLink on the web or desktop Standard Notes application to uninstall then reinstall this sync provider.

But clicking on the instructions link I can find following information:

> This feature is no longer available. There are still many great ways to get backups of your account data, including daily email backups, automatic desktop backups, and backups you download directly via the desktop, web, or mobile application.

WTF?

Why are you removing Google Drive? Email backups are really not a nice way to organize backups (I use them as backup fallback, but none the less this is really a strange decision)?

Btw, I'm a paying user.

**Comment by effieeee** [comment]
Hi @h0jeZvgoxFepBQ2C, you may not have received the email notice yet, but here is a copy of it, which briefly explains why he had to remove this functionality of the app:

> Hello,

> 
> This message is to inform you that due to relatively low adoption and increasingly complex APIs necessary to integrate with multiple third party cloud providers, we will no longer be offering our daily backup service to third party cloud providers such as Dropbox, Google Drive, and OneDrive.

> 
> There are still many great ways to get backups of your account data, including daily email backups, automatic desktop backups, and backups you download directly via the desktop, web, or mobile application.

> 
> For more information on how to create and import account backups, visit our [help center](https://standardnotes.com/help/14/how-do-i-create-and-import-backups-of-my-standard-notes-data).
